jest-setup.ts

```diff
@@ -38,4 +38,17 @@ jest.mock( 'strip-ansi', () => ( {
 	default: jest.fn().mockImplementation( ( str: string ) => str ),
 } ) );
 
+jest.mock( './src/hooks/use-ai-icon', () => ( {
+	__esModule: true,
+	default: jest.fn().mockReturnValue( {
+		rive: jest.fn(),
+		RiveComponent: jest.fn(),
+		inactiveInput: jest.fn(),
+		typingInput: jest.fn(),
+		thinkingInput: jest.fn(),
+		startStateMachine: jest.fn(),
+		pauseStateMachine: jest.fn(),
+	} ),
```

Comment:
I think we don't need to mock the `default` function unless we expect to mock it in test cases. Hence, it should be enough with:
```suggestion
	default: () => ( {
		rive: jest.fn(),
		RiveComponent: jest.fn(),
		inactiveInput: jest.fn(),
		typingInput: jest.fn(),
		thinkingInput: jest.fn(),
		startStateMachine: jest.fn(),
		pauseStateMachine: jest.fn(),
	} ),
```

Reply:
Yep. The tests still pass with this approach. Thanks @fluiddot